core/src/system_monitor_service.rs

```diff
@@ -15,7 +15,7 @@ use {
 };
 
 const MS_PER_S: u64 = 1_000;
-const SAMPLE_INTERVAL_UDP_MS: u64 = 60 * MS_PER_S;
+const SAMPLE_INTERVAL_UDP_MS: u64 = MS_PER_S;
```

Comment:
What's the motivation for reducing the interval?

Reply:
I changed this to 2s to have parity with other sampling. I was thinking that 60s could mask spikes when taken across many hosts.

Reply:
got it!  sgtm!